scene/3d/bone_attachment_3d.cpp

```diff
@@ -377,4 +422,12 @@ void BoneAttachment3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::STRING_NAME, "bone_name"), "set_bone_name", "get_bone_name");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "bone_idx"), "set_bone_idx", "get_bone_idx");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "override_pose"), "set_override_pose", "get_override_pose");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "bone_pose_mode", PROPERTY_HINT_ENUM, "Bone Pose,Bone Pose Override,Bone Pose No Override, Bone Rest"), "set_bone_pose_mode", "get_bone_pose_mode");
+
+	BIND_ENUM_CONSTANT(BONE_POSE);
+	BIND_ENUM_CONSTANT(BONE_POSE_OVERRIDE);
+	BIND_ENUM_CONSTANT(BONE_POSE_NO_OVERRIDE);
+	BIND_ENUM_CONSTANT(BONE_REST);
 }
+
+VARIANT_ENUM_CAST(BoneAttachment3D::BonePoseMode);
```

Comment:
```suggestion
VARIANT_ENUM_CAST(BoneAttachment3D::BonePoseMode);

```
Missing trailing line termination